dev-docs/howto/vpn/helm/README.md

```diff
@@ -2,35 +2,84 @@
 
 This Helm chart deploys a VPN server to your Constellation cluster.
 
-## Installation
+## Prerequisites
 
-1. Create and populate the configuration.
+* Constellation >= v2.14.0
+* A publicly routable VPN endpoint on premises that supports any of
+  * IPSec in IKEv2 tunnel mode with NAT traversal enabled
+  * Wireguard
```

Comment:
```suggestion
```
I think we need to drop that in this PR.

Reply:
I reworded the prerequisite section and removed references to Wireguard from `values.yaml`, too.